apps/prairielearn/src/pages/workspace/workspace.html.ts

```diff
@@ -84,6 +100,18 @@ export function Workspace({
               <li class="d-sm-none nav-item ml-2 my-1">
                 <span class="nav-item badge badge-light">${navTitle}</span>
               </li>
+              ${resLocals.assessment &&
+              resLocals.assessment.type === 'Exam' &&
```

Comment:
Nit, slightly more concise:

```suggestion
              ${resLocals.assessment?.type === 'Exam' &&
```

Reply:
Changed.